src/Sulu/Bundle/TagBundle/Resources/config/services.xml

```diff
@@ -3,43 +3,30 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
-        <parameter key="sulu_tag.admin.class">Sulu\Bundle\TagBundle\Admin\TagAdmin</parameter>
-        <parameter key="sulu_tag.tag_manager.class">Sulu\Bundle\TagBundle\Tag\TagManager</parameter>
-        <parameter key="sulu_tag.tag_repository.class">Sulu\Bundle\TagBundle\Entity\TagRepository</parameter>
         <parameter key="sulu_tag.content.type.tag_list.class">Sulu\Bundle\TagBundle\Content\Types\TagList</parameter>
-        <parameter key="sulu_tag.tag_request_handler.class">Sulu\Component\Tag\Request\TagRequestHandler</parameter>
-        <parameter key="sulu_tag.twig_extension.class">Sulu\Bundle\TagBundle\Twig\TagTwigExtension</parameter>
-        <parameter key="sulu_tag.entity.tag">SuluTagBundle:Tag</parameter>
     </parameters>
     <services>
-        <service id="sulu_tag.admin" class="%sulu_tag.admin.class%">
+        <service id="sulu_tag.admin" class="Sulu\Bundle\TagBundle\Admin\TagAdmin">
             <tag name="sulu.admin"/>
             <tag name="sulu.context" context="admin"/>
             <argument type="service" id="sulu_security.security_checker"/>
         </service>
-        <service id="sulu_tag.tag_repository" class="%sulu_tag.tag_repository.class%">
-            <factory method="getRepository" service="doctrine.orm.entity_manager"/>
-            <argument>%sulu_tag.entity.tag%</argument>
-        </service>
-        <service id="sulu_tag.tag_manager" class="%sulu_tag.tag_manager.class%" public="true">
+        <service id="sulu_tag.tag_manager" class="Sulu\Bundle\TagBundle\Tag\TagManager" public="true">
```

Comment:
Does it need to be public? From my understanding we were only doing this for the services that absolutely need it.

Reply:
Was public before (little tricky in this diff as I removed the repository).
Currently it needs to be public as it is used in `TagController` and `SmartContentItemController`